uast/uast.go

```diff
@@ -82,7 +82,7 @@ type Position struct {
 	Offset uint32 `json:"offset"`
 	// Line is the line number. It is a 1-based index.
 	Line uint32 `json:"line"`
-	// Col is the column number (the byte offset of the position relative to
+	// Col is the column number - the byte offset of the position relative to
```

Comment:
Is the "-" here meant to be a dash or a minus?

If a dash, please use either "—" or "--" to make it clear (since if it's subtraction, the description has become wrong I think).